_posts/2018-08-31-zenika-craftsman-swap.md

```diff
@@ -0,0 +1,78 @@
+---
+author: Christian Panadero Martinez
+layout: post
+asset-type: post
+title: "Zenika craftsman swap"
+date: 2018-08-31 08:01:00
+description: Summary and lessons learned during my craftsman swap in Paris
+image:
+    src: /assets/custom/img/blog/2018-08-31-zenika-craftsman-swap/znk.png
+tags:
+- software processes
+- software craftsmanship
+- agile
+- xp
+---
+The past week of the 6th of August I did a craftsman swap and I would like to share how was my experience, the lessons that I've learned and how fun it was.
```

Comment:
Sorry for the late review, but I was just reading the article and spotted that this sentence is a near-duplicate of the following sentence. It looks as if it was refactored at some point but the previous sentence wasn't deleted?